nano/node/vote_processor.cpp

```diff
@@ -202,29 +202,17 @@ nano::vote_code nano::vote_processor::vote_blocking (nano::transaction const & t
 	auto result (nano::vote_code::invalid);
 	if (validated || !vote_a->validate ())
 	{
+		auto is_replay (active.vote (vote_a, true));
+		result = boost::logic::indeterminate (is_replay) ? nano::vote_code::indeterminate : is_replay ? nano::vote_code::replay : nano::vote_code::vote;
```

Comment:
Could `vote` return `nano::vote_code` directly?

Reply:
Definitely, thanks for noticing. Made `active::vote` much more evident in what the return is; tribool is useful but not in this case where we want `indetermined || false` to be false.